goals.md

```diff
@@ -0,0 +1,6 @@
+
+# Goals
+
+ 1. Address the issues of Celery
```

Comment:
This should not be a goal.  Issues with Celery should be fixed in Celery, not here.